catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Segment.js

```diff
@@ -26,13 +26,21 @@ define([
 ], function (Backbone, _, FieldDescriptors, Field) {
 
     var counter = 0;
-    var typeCounters= {
+    var typeCounters = {
         Service: 1,
         Organization: 1,
         Content: 1,
         ServiceBinding: 1
     };
 
+    function checkDateValidity(values) {
+        var dateVal = new Date(values);
+        if (!isNaN(dateVal.getTime())) {
```

Comment:
I think this runs into a similar issue as above. If i use _.isDate, it will always be true, even if invalid.